pkg/projects/Microsoft.Net.UWPCoreRuntimeSdk/files/Microsoft.Net.UWPCoreRuntimeSdk.props

```diff
@@ -0,0 +1,5 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <UseAppLocalCoreCLR>true</UseAppLocalCoreCLR>
+  </PropertyGroup>
```

Comment:
I am still a bit lost ..i don't see this props file being imported anywhere. is that not required?